hub/codec/__init__.py

```diff
@@ -0,0 +1,2 @@
+from hub.codec.numcodecs import NumPy, Lz4, Zstd, BaseNumCodec
+from hub.codec.image_codecs import PngCodec, JpegCodec, WebPCodec, BaseImgCodec
```

Comment:
another note: can you not name these classes with different conventions?

i really like the idea of having `PNG` and `JPEG` instead of the "codec" prefixes